db/migrate/20150427100026_add_devise_recoverable_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeviseRecoverableToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :reset_password_token, :string   
```

Comment:
Should there be an index? I notice there is one for the confirmation token in the schema.